datafusion/core/src/physical_optimizer/pruning.rs

```diff
@@ -1958,6 +1964,90 @@ mod tests {
         Ok(())
     }
 
+    #[test]
+    fn row_group_predicate_between() -> Result<()> {
+        let schema = Schema::new(vec![
+            Field::new("c1", DataType::Int32, false),
+            Field::new("c2", DataType::Int32, false),
+        ]);
+
+        // test c1 BETWEEN 1 AND 5
+        let expr1 = Expr::Between(Between {
+            expr: Box::new(col("c1")),
+            negated: false,
+            low: Box::new(lit(1)),
+            high: Box::new(lit(5)),
+        });
```

Comment:
You could also use `between` to make this more concise: https://docs.rs/datafusion/latest/datafusion/logical_expr/enum.Expr.html#method.between like:

```suggestion
        let expr1 = col("c1").between(lit(1), lit(5));
```

Reply:
thanks!